packages/components/src/components/hds/dropdown/index.ts

```diff
@@ -44,6 +44,7 @@ export interface HdsDropdownSignature {
     listPosition?: HdsDropdownPositions;
     width?: string;
     enableCollisionDetection?: FloatingUIOptions['enableCollisionDetection'];
+    preserveDomContent?: boolean;
```

Comment:
what do you think of this name? does it convey what its intent is? do you have other ideas/suggestions?

Reply:
What do you think about `persistOptionsInDom`?

Reply:
they are not options :( 
they are not even "items" because this includes also the header and footer elements
maybe `persistDomContent`?